Library/ViewModels/PledgeViewModel.swift

```diff
@@ -52,12 +110,20 @@ public class PledgeViewModel: PledgeViewModelType, PledgeViewModelInputs, Pledge
     self.configureProjectAndRewardProperty.value = (project, reward)
   }
 
+  private let reloadDataProperty = MutableProperty<Void>(())
```

Comment:
Did we need this in the end? I thought that other solution worked 😓

Reply:
If we have no alternative I'd call this `didReloadData` just for some additional clarity.

Reply:
This was the only way to get it reliably working 😢 the other solution we had only worked sometimes - it was a race condition after all. This feels like a better solution anyway, because we can guarantee that `tableView.reloadData()` has been called.